### PR TITLE
Network additions

### PR DESCRIPTION
**Pull request by aholachek**
The main changes included 
- adding a graph to show the times that different groups were active, measured by the amount of papers they produced
- adding an ability to "hop" from author to author in the author network, with a back button that allows the user to go to the previous network
- standardizing author and paper network to use bootstrap layout classes so they are responsive, and just improving the styles in general

### CODE REVIEW

**Comment by romanchyla** [comment]
the summary view seems to be very very tiny (i've regenerated less,  made no difference)

screenshot - 04102015 - 04 01 02 pm

and the question marks disappeared (instead it points to a help page) - i think that is a regression, since the user will leave the page. could we have the question marks (and link to the help pages only from the short help?) i belive this will give you also more space to use

btw: this PR would benefit from descriping the nature of the changes, it is not obvious from the code

File: test/mocha/js/widgets/network_widget.spec.js [hunk 1]
```diff
@@ -1758,6 +1773,9 @@ define([
     networkWidget.view.graphView.model.set("selectedEntity", accomazziNode);
     $("#test").find("button.filter-add").click();
 
+
+    debugger;
```
Comment (romanchyla) [inline test/mocha/js/widgets/network_widget.spec.js]: tbremoved

**Comment by romanchyla** [comment]
Alex, can you pls rebase this one? thanks

**Comment by aholachek** [comment]
Done!

**Pull request merged**